docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
+```
+AzurePublicCloudHostUrl = "https://login.microsoftonline.com/";
+AzureChinaHostUrl = "https://login.chinacloudapi.cn/";
+AzureGermanyHostUrl = "https://login.microsoftonline.de/";
+AzureGovernmentHostUrl = "https://login.microsoftonline.us/";
+```
+2. You can specify the `MICROSOFT_GRAPH_API_HOST` URLs based you your environment:
+```plaintext
```

Comment:
@colefreeman You can remove `plaintext` from here as well. If I make a comment on something in one place, it implies anywhere else in the file where the same situation occurs.

Reply:
Addressed and tested in mintlify dev env.